package.json

```diff
@@ -11,10 +11,16 @@
   "license": "ISC",
```

Comment:
under scripts you could add one for test and instead of node you do mocha .\app\server-test